src/algorithms/utils/TimeSeries.test.ts

```diff
@@ -1,6 +1,106 @@
 import { uniformDatesBetween, makeTimeSeries, updateTimeSeries } from './TimeSeries'
 import { interpolateTimeSeries } from '../run'
 
+import * as fs from 'fs'
+const { utils } = require('jest-snapshot')
+
+expect.extend({
+  toBeCloseToArraySnapshot(received: number[]) {
+    const { testPath, currentTestName, snapshotState } = this
+    snapshotState._counters.set(currentTestName, (snapshotState._counters.get(currentTestName) || 0) + 1)
+    const count = Number(snapshotState._counters.get(currentTestName))
```

Comment:
This code was lifted from Jest's `toMatchSnapshot()` implementation. Not entirely sure what it's doing, but the test doesn't work without this.